Header header
int16 TEMP_MOS1
int16 TEMP_MOS2
int16 TEMP_MOS3
int16 TEMP_MOS4
int16 TEMP_MOS5
int16 TEMP_MOS6
int16 TEMP_PCB
int32 current_motor
int32 current_in
int16 duty_now
int32 rpm
int32 v_in
int32 amp_hours
int32 amp_hours_charged
int32 watt_hours
int32 watt_hours_charged
int32 tachometer
int32 tachometer_abs
float32 current_steering_angle
float32 current_speed
